pkg/middlewares/route53_records.go

```diff
@@ -30,7 +30,7 @@ func (m Route53DefaultZoneRecordSanitizer) Execute(remoteResources, resourcesFro
 
 		record, _ := remoteResource.(*aws.AwsRoute53Record)
 
-		if !isDefaultRecord(record) {
+		if !isDefaultRoute53Record(record) {
```

Comment:
Is this related to this PR ?

Reply:
Yes, `isDefaultRecord` was too generic and not specific to Route53. As I created a similar method for various resources, I needed to rename this method so it does not conflict with other methods in the same package.

Reply:
Why not, I'm not against it. But indeed, it is not related to this PR.

What is the other method that conflict with this one ?

Reply:
The method is not used anymore, so I reverted this change